app/selectors/transaction.js

```diff
@@ -0,0 +1,8 @@
+import { reduce } from 'lodash';
+import { createSelector } from 'reselect';
+
+const getTransactionItem = (state) => (state.transaction_item);
+
+export const transactionAmountSelector = createSelector(getTransactionItem, (items) => (
+  reduce(items, (memo, item) =>(memo + item.value * item.quantity), 0)
+));
```

Comment:
@elvongray yes, so I will remove reselect but still keep the selector, is it ok?